src/components/Posts/LikeSection.js

```diff
@@ -8,23 +8,28 @@ import { faComment, faHeart } from '@fortawesome/free-regular-svg-icons';
 
 const LikeSection = props => {
   // 🔥 Make sure the parent of LikeSection is passing the right props!
-  const { likePost, numberOfLikes } = props;
+  const { post, likePost, numberOfLikes } = props;
 
   return (
     <div>
       <div
         className='like-section'
         key='likes-icons-container'
       >
-        <div className='like-section-wrapper'>
+        <div className='like-section-wrapper' onClick={(click) => {
```

Comment:
This will only increase the like once. You might want to make a function to call within the onClick that can be called multiple times.